### ISSUE TITLE
weighting property on relationship?

### ISSUE THREAD

**Issue by Lkruitwagen** [opensupplyhub/supplychaindata.exchange#13]
For some use cases we want more data/properties on `Relationships` right? E.g. % shareholding for ownership, total supplier volume in a given year, etc.

Is this the kind of thing that would go into an extension?

**Comment by wonderchook** [comment]
Yes, we are intentionally keeping the core very light but additional information could go into an extension.
